client/containers/DashboardSubmissions/SubmissionItems.tsx

```diff
@@ -112,9 +121,30 @@ const SubmissionItems = (props: Props) => {
 				filters={overviewSearchFilters}
 			/>
 			<OverviewRows>
-				{renderPubs()}
+				{(pubs as DefinitelyHas<Pub, 'submission'>[])
```

Comment:
Could we make `DefinitelyHas<Pub, 'submission'>[]` the type of the `pubs` prop?

Reply:
don't think so? We're pulling it out of both `useManyPubs` and `props.initialPubs`. I'm down to put it to `props.initialPubs`, but I don't understand how we could expect presence of submissions on pubs returned from `usemanyPubs`

Reply:
nevermind, TS ain't complaining so neither am I.